tests/test-build.js

```diff
@@ -1,35 +1,47 @@
 /* eslint-env node */
-const { readFile, lstat } = require("fs").promises;
+const {
+  constants: { F_OK },
+  promises: { readFile, access },
+} = require("fs");
+const { execSync } = require("child_process");
 const path = require("path");
-const expect = require("chai").expect;
 const { Builder } = require("../tools/builder");
 
 async function fileExists(filePath) {
-  const stats = await lstat(filePath);
-  return stats.fileExists();
+  try {
+    await access(filePath, F_OK);
+    return true;
+  } catch {
+    return false;
+  }
 }
 
 describe("builder (tool)", () => {
   jasmine.DEFAULT_TIMEOUT_INTERVAL = 60000;
 
   const profiles = ["w3c", "geonovum"];
+  const rootDir = path.join(__dirname, "..");
 
   beforeAll(async () => {
     await Promise.all(
       profiles.map(profile => Builder.build({ name: profile }))
     );
   });
 
+  afterAll(() => {
+    execSync("git checkout -- builds", { cwd: rootDir });
```

Comment:
`checkout` for this purpose is considered as legacy, how about:

```suggestion
    execSync("git restore builds", { cwd: rootDir });
```

Reply:
Depends on whether @marcoscaceres has updated his git :smile:

Reply:
`sudo login to marcos's computer && sudo apt upgrade everything`

Reply:
My git is always up to date :)